corpus/pages/views.py

```diff
@@ -42,5 +46,40 @@ def sig(request, sig_name):
 
     return render(request, "pages/sig.html", args)
 
+
+def team(request):
+    compsoc_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
```

Comment:
I don't think you've made `ExecutiveMember.core` as an association right? I think this is what this query does?

Reply:
I have made a Core.executivemember relation. Again since every Core member has a user model associated, i'm just checking that we do not fetch such users